pkg/kubebench/scanner_test.go

```diff
@@ -0,0 +1,221 @@
+	"github.com/aquasecurity/starboard/pkg/ext"
+	"github.com/aquasecurity/starboard/pkg/kubebench"
+	"github.com/aquasecurity/starboard/pkg/starboard"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+)
+
+var (
+	fixedTime  = time.Now()
+	fixedClock = ext.NewFixedClock(fixedTime)
+)
+
+func TestKubeBenchPlugin_GetScanJobSpec(t *testing.T) {
+	config := starboard.ConfigData{
+		"kube-bench.imageRef": "docker.io/aquasec/kube-bench:0.5.0",
+	}
+	node := corev1.Node{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "control-plane",
+		},
+	}
+	instance := kubebench.NewKubeBenchPlugin(fixedClock, config)
+
+	podSpec, err := instance.GetScanJobSpec(node)
+
+	require.NoError(t, err)
+	assert.Equal(t, corev1.PodSpec{
+		ServiceAccountName:           starboard.ServiceAccountName,
+		AutomountServiceAccountToken: pointer.BoolPtr(true),
+		RestartPolicy:                corev1.RestartPolicyNever,
+		HostPID:                      true,
+		NodeName:                     node.Name,
+		SecurityContext: &corev1.PodSecurityContext{
+			RunAsUser:  pointer.Int64Ptr(0),
+			RunAsGroup: pointer.Int64Ptr(0),
+			SeccompProfile: &corev1.SeccompProfile{
+				Type: corev1.SeccompProfileTypeRuntimeDefault,
+			},
+		},
+		Volumes: []corev1.Volume{
+			{
+				Name: "var-lib-etcd",
+				VolumeSource: corev1.VolumeSource{
+					HostPath: &corev1.HostPathVolumeSource{
+						Path: "/var/lib/etcd",
+					},
+				},
+			},
+			{
+				Name: "var-lib-kubelet",
+				VolumeSource: corev1.VolumeSource{
+					HostPath: &corev1.HostPathVolumeSource{
+						Path: "/var/lib/kubelet",
+					},
+				},
+			},
+			{
+				Name: "etc-systemd",
+				VolumeSource: corev1.VolumeSource{
+					HostPath: &corev1.HostPathVolumeSource{
+						Path: "/etc/systemd",
+					},
+				},
+			},
+			{
+				Name: "etc-kubernetes",
+				VolumeSource: corev1.VolumeSource{
+					HostPath: &corev1.HostPathVolumeSource{
+						Path: "/etc/kubernetes",
+					},
+				},
+			},
+			{
+				Name: "usr-bin",
+				VolumeSource: corev1.VolumeSource{
+					HostPath: &corev1.HostPathVolumeSource{
+						Path: "/usr/bin",
+					},
+				},
+			},
+		},
+		Containers: []corev1.Container{
+			{
+				Name:                     "kube-bench",
+				Image:                    "docker.io/aquasec/kube-bench:0.5.0",
+				ImagePullPolicy:          corev1.PullIfNotPresent,
+				TerminationMessagePolicy: corev1.TerminationMessageFallbackToLogsOnError,
+				Command:                  []string{"sh"},
+				Args:                     []string{"-c", "kube-bench --json 2> /dev/null"},
+				SecurityContext: &corev1.SecurityContext{
+					Privileged:               pointer.BoolPtr(false),
+					AllowPrivilegeEscalation: pointer.BoolPtr(false),
+					Capabilities: &corev1.Capabilities{
+						Drop: []corev1.Capability{"all"},
```

Comment:
```suggestion
						Drop: []corev1.Capability{"ALL"},
```

Isn't this supposed to be ALL (caps)?

According to [spec](https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted), PSA implementation, and most policy rules?